docs/vignettes/themes.qmd

```diff
@@ -0,0 +1,167 @@
+# Themes and Customization
+
+The `altdoc/` directory is created when you call `setup_docs()`. It stores the basic settings that `altdoc` uses to render the documentation website. To customize your documentation website, you can edit the settings files stored in the `altdoc/` directory before calling `render_docs()`. 
+
+Note that the contents of `altdoc/` are never modified when you update the website with `render_docs()`. Therefore, you can safely edit the settings files by hand to customize your website. (Warning: `altdoc/` can be overwritten if you call `setup_docs()` again with the `overwrite=TRUE` argument.)
+
+Each of the documentation engines that `altdoc` supports offers extensive theming and customization options. This vignette illustrates the process using minimal examples for each engine, and provides links to the relevant documentation of each engine for users who want to dive deeper.
+
+## Docsify
+
+Docsify supports many themes and offers extensive configuration possibilities. Here, we show a minimal example using the Themeable framework. For more information, see:
+
+* [Docsify User Guide](https://docsify.js.org/#/)
+* [Themeable for Docsify](https://jhildenbiddle.github.io/docsify-themeable/#/)
+
+Initialize the documentation website settings:
+
+```r
+library(altdoc)
+setup_docs("docsify")
+```
+
+Edit the `altdoc/docsify.html` file. For example, to use the Themeable framework for Docsify, we only have to insert two lines in the settings file.
+
+First, we replace the default Docsify theme with:
+
+```html
+<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/docsify-themeable@0/dist/css/theme-simple.css">
+```
+
+Then, after the Docsify script, we insert:
+
+```html
+<script src="https://cdn.jsdelivr.net/npm/docsify-themeable@0/dist/js/docsify-themeable.min.js"></script>
+```
+
+The full modified HTML template is shown below.
+
+Render the documentation:
+
+```r
+render_docs()
+```
+
+Here is a preview of the result:
+
+![](images/themes_docsify_preview.png)
+
+
+Click on Details to display the full HTML template:
+
+<details>
```

Comment:
I think it also requires a `<summary>` tag for the text of the button, like "click here"

https://www.w3schools.com/TAGS/tag_details.asp

Reply:
really not sure about this actually

Reply:
I rendered it and what this shows currently is the text above, and then on the next line you have an arrow with the word "Details".

I think it's pretty clear currently, but I'm not an HTML guru.

Reply:
Ah ok, fine then, I just didn't see the rendered output